Planner/src/Planner.java

```diff
@@ -122,8 +157,11 @@ public void actionPerformed(ActionEvent arg0) {
 			    taskPanel.add(assignedTo);
 			    taskPanel.add(new JLabel("Start time. Enter yyyy-MM-dd HH:mm:ss")); 
 			    taskPanel.add(start); // change to field where user can select the date and time
+			    taskPanel.add(datePanel);
 			    taskPanel.add(new JLabel("End time. Enter yyyy-MM-dd HH:mm:ss"));
 			    taskPanel.add(end); // change to field where user can select the date and time
+			    taskPanel.add(datePanel);
```

Comment:
We don't have to add twice datePanel, we need to create another 3 JComboBoxes for endTime as well and maybe add them in anoter panel (maybe have a startPanel and endPanel). Also you can add combo boxes for hours and minutes. For seconds I think it's not necessary, we will also put them to 0.